dev/src/v1/firestore_client.ts

```diff
@@ -955,6 +956,96 @@ export class FirestoreClient {
     this.initialize();
     return this.innerApiCalls.createDocument(request, options, callback);
   }
+  batchWrite(
```

Comment:
We can't make this public. Thoughts on how to resolve this?

Reply:
Deleted the `batchWrite` code in `firestore_client` and its corresponding usages in `gapic_firestore_v1.ts`.